virtualcluster/pkg/controller/controllers/virtualcluster_controller_test.go

```diff
@@ -179,6 +179,7 @@ var _ = Describe("VirtualCluster Controller", func() {
 			}, time.Minute*5, interval).Should(BeTrue())
 
 			By("Updating ClusterVersion")
+			Expect(cli.Get(ctx, client.ObjectKeyFromObject(cvInstance), cvInstance)).Should(Succeed())
```

Comment:
@Fei-Guo  Could please take a look and trigger the test again?

I found the test case has this error message:
```
Operation cannot be fulfilled on clusterversions.tenancy.x-k8s.io "clusterversion-samplegh6xl": the object has been modified; please apply your changes to the latest version and try again
```
IMHO, the reason for the error is because the `cvInstance` object is staled here. So here I am trying to re-get the object. It works on my local environment.
But this error doesn't seem to have happened in the past, which is a little bit weird.